src/helpers/vyos-check-wwan.py

```diff
@@ -29,6 +30,13 @@
             # do not restart this interface as it's disabled by the user
             continue
 
+        # user wish to use ping to validate interface
+        if 'enable' in interface_config.get("ping-check"):
```

Comment:
Will `.get` not raise an exception if ping-check dict is not defined?

Also this script does not handle default values, which you have presented in xml.

Reply:
```python
>>> None.get()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
```